src/_common/media-item/media-item-model.ts

```diff
@@ -42,6 +42,8 @@ export class MediaItem extends Model {
 	mediaserver_url_webm!: string;
 	mediaserver_url_mp4!: string;
 	mediaserver_url!: string;
+	avg_img_color!: string;
```

Comment:
I forgot to tell you. This property should be `null | string`. We dont' ahve the data on old media items. You'll have to update the itemStyling() appropriately, to only apply the background color when this property is passed in